app/commands/welcome.js

```diff
@@ -0,0 +1,175 @@
+module.exports = Base => class extends Base {
+    constructor() {
+        super();
+        //this allows us to sort the welcome array in the player schema alphabetically and keep default as the first index.
+        this.welcomeOptions = {
+            'default': 'a',
+            'online': 'b',
+            'help': 'c',
+            'cooldowns': 'd',
+            'autostore': 'e',
+            'version': 'f',
+            'hints': 'z',
+            'reset': 'reset'
+        };
+
+        this.serverVersion = [
+            { text: 'version: 0.66', color: 'green' },
+            '\nThis update adds custom welcome messages. type ',
+            { text: '!help welcome', color: 'green' },
+            ' for more info.\nThis includes a handful of hints aimed to help players discover new features.'
+        ];
+
+        this.helpShortDescription.welcome = [
+            'Customize the message that greets you when you login. ex: ',
+            { text: '!welcome cooldowns', color: 'green' },
+        ];
+
+        this.helpFullDescription.welcome = [
```

Comment:
This creates a dependency for every other file on this file. Not sure I like how this is setup

Reply:
I've edited the description to make it much shorter. as for your worry about dependency i think that comes from a misunderstanding of what was communicated. the only things listed are options for the welcome message. functions separate from other files. you worried about adding future commands and how that would affect this, and the answer is that it wouldn't, unless we specifically decided we wanted a future welcome message option related to it.